runtime/dart_vm.cc

```diff
@@ -470,14 +470,14 @@ DartVM::DartVM(std::shared_ptr<const DartVMData> vm_data,
     // Use a duration event so about:tracing will consider this event when
     // deciding the earliest event to use as time 0.
     if (settings_.engine_start_timestamp.count()) {
-      Dart_TimelineEvent(
-          "FlutterEngineMainEnter",                  // label
-          settings_.engine_start_timestamp.count(),  // timestamp0
-          Dart_TimelineGetMicros(),                  // timestamp1_or_async_id
-          Dart_Timeline_Event_Duration,              // event type
-          0,                                         // argument_count
-          nullptr,                                   // argument_names
-          nullptr                                    // argument_values
+      int64_t micros = Dart_TimelineGetMicros();
+      Dart_TimelineEvent("FlutterEngineMainEnter",     // label
+                         micros,                       // timestamp0
```

Comment:
How does this affect the benchmark value reported for non-Windows?

Does the comment above need to be adjusted?

Reply:
It will shift the earliest timestamp forward by the time taken to initialize the VM. This seems a preferable outcome to having inconsistent and sometimes meaningless timestamps. The comment should be changed as it is not using a duration any more.

Reply:
@dnfield Will this affect the startup metrics used by `customer: money`?

Reply:
I don't think so but @jiahaog would know for sure

Reply:
This will affect 1P startup metrics that read from the observatory (in profile mode), such as from Flutter driver. We read only the `ts` value of this metric, which is the `timestamp0` parameter. 

I don't think this is a blocker though, as customers can rebaseline their tests. We have also been directing customers to move away from such metrics to more general Android ones, and `customer: money` specifically does not use this metric.

Reply:
Do you then see a need for further changes to this PR?

Reply:
No, this should be fine. Thanks :)